demos/ota/ota_demo_core_http/ota_demo_core_http.c

```diff
@@ -1408,24 +1377,21 @@ static uint32_t prvGetTimeMs( void )
 
     return ulTimeMs;
 }
-
-
-
 /*-----------------------------------------------------------*/
 
-static MQTTStatus_t prvMqttInit( void )
+static MQTTStatus_t prvMqttAgentInit( void )
 {
     TransportInterface_t xTransport;
     MQTTStatus_t xReturn;
     MQTTFixedBuffer_t xFixedBuffer = { .pBuffer = xNetworkBuffer, .size = MQTT_AGENT_NETWORK_BUFFER_SIZE };
-    static uint8_t staticQueueStorageArea[ MQTT_AGENT_COMMAND_QUEUE_LENGTH * sizeof( Command_t * ) ];
-    static StaticQueue_t staticQueueStructure;
+    static uint8_t usStaticQueueStorageArea[ MQTT_AGENT_COMMAND_QUEUE_LENGTH * sizeof( Command_t * ) ];
```

Comment:
This should start with "uc" instead of "us" since this is an 8 bit variable.

Reply:
Done https://github.com/aws/amazon-freertos/pull/3148/commits/6bbe8a3c217e31fff82104a685ad811120104724